app/partials/subject-uploader.cjsx

```diff
@@ -66,7 +66,9 @@ module.exports = React.createClass
         .then (subject) =>
           uploads = for typeToUploadURL, i in subject.locations
             uploadURL = typeToUploadURL[Object.keys(typeToUploadURL)[0]]
-            putFile uploadURL, @props.files[subjectData.locations[i]]
+            headers =
+              'Content-Type': locationTypes[i]
+            putFile uploadURL, @props.files[subjectData.locations[i]], headers
```

Comment:
We'll need to add this in the other places putFile is used, e.g. avatar/background upload etc 
https://github.com/zooniverse/Panoptes-Front-End/search?utf8=%E2%9C%93&q=putFile